bluesky_widgets/models/plot_builders.py

```diff
@@ -777,6 +785,10 @@ def __init__(
         needs_streams=("primary",),
         namespace=None,
         axes=None,
+        clim=None,
+        cmap='viridis',
+        extent=None,
+        aspect='equal',
```

Comment:
After talking through the various trade-offs yesterday, I came away with the feeling that we should remove this parameter entirely (along with the associated `property`) and leave it to the user to do this via either preconstructing `AxesSpec` manually or modifying it afterward directly.

```py
# Manually create Axes so as to set custom aspect.
axes = AxesSpec(aspect="...")  # might also set custom title or whatever they like...
rastered_image = RasteredImage(..., axes=axes)

# Or accept the default Axes but modify them after creation.
rastered_image = RasteredImage(...)
rastered_image.axes.aspect = "..."
```

My reasoning is that if we give special access to `aspect` here, why not also `title` and `x_label` and `y_label`.... it gets quickly messy.

In the block below in `__init__` where we auto-create at `AxesSpec` if one is not specifically given to us, we should be opinionated and pass in `aspect="equal"` since that is a more suitable default for `RasteredImage`.

Reply:
Yep, I agree. I didn't make note of what we decided in my notes so I forgot about it